src/_types.ts

```diff
@@ -1,9 +1,15 @@
+export enum TimestampFormat {
```

Comment:
We may want to re-export `TimestampFormat` from `index.tsx` as we have done with many of the query request/response types; I think this may allow consumers to import these types using 
```ts
import { FooType } from "@chalk-ai/client";
```
rather than from an internal types file, which they prefer (see https://chalk-ai.slack.com/archives/C049B5VDRMZ/p1715247294929619).

Since this is a type that is meant to be imported and used by customers it may also make more sense to locate this type in `_interface.ts`, but as we discussed earlier I don't have a strong opinion about conforming to that pattern.